.github/workflows/ci.yml

```diff
@@ -0,0 +1,80 @@
+name: GH Actions CI
+
+on:
+  # run every Sunday at 3:53 UTC
+  schedule:
+    - cron: "53 3 * * 0"
+  push:
+    branches:
+      - "master"
+  pull_request:
+    branches:
+      - "master"
+
+concurrency:
+  group: "${{ github.ref }}-${{ github.head_ref }}"
+  cancel-in-progress: true
+
+jobs:
+  unittests:
+    name: Test ${{ matrix.os }} with Python ${{ matrix.python-version }}
+    if: github.repository == 'MDAnalysis/MDAnalysisData'
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+        python-version: [2.7, 3.6, 3.7, 3.8, 3.9]
+        exclude:
+          - os: macOS-latest
+            python-version: 3.6
+          - os: macOS-latest
+            python-version: 3.7
+          - os: macOS-latest
+            python-version: 3.9
+          - os: windows-latest
+            python-version: 3.6
+          - os: windows-latest
+            python-version: 3.7
+          - os: windows-latest
+            python-version: 3.9
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup python
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Display Python version
+      run: python -c "import sys; print(sys.version)"
+
+    - name: Install pytest, pytest plugins
+      run: |
+        python -m pip install wheel
+        python -m pip install pytest pytest-cov pytest-pep8 pytest-mock codecov
+
+    - name: Install main dependencies
```

Comment:
any reason for having the two separate parts of the workflow and not just squashing everything into the one step?

Reply:
Mainly because I like the semantics of clearly seeing where something for tests only is done vs what's needed for running the package — I often look at the CI files to figure out what's _actually_ needed as opposed to what someone might or might not have put into `setup.py`.